src/h2load.cc

```diff
@@ -271,9 +345,32 @@ void Client::disconnect() {
 }
 
 void Client::submit_request() {
+  restart_timeout();
```

Comment:
So we need to re-arm timer on every network activity.  Then, it is simpler to do this in writecb and readcb above, instead of calling restart_timeout() in every single hook functions.